### ISSUE TITLE
Feature request: download all attachments

### ISSUE THREAD

**Issue by errant-info** [Bannerets/keybase-export#17]
Would love a feature that allows downloading all chat attachments

There's some guidance on reddit here:
https://www.reddit.com/r/Keybase/comments/hp2izo/how_to_download_attachment_using_command_line/

Unsure if there's already an attachments option available in the config.json that will do this?

**Comment by eilvelia** [comment]
It's possible to add attachment downloading (and was in TODO from the start), but requires some work. I'll see when I can work on that. Currently files can be downloaded manually by using that `keybase chat download` command with ids of messages that contain the `attachment` field in the export.

**Comment by eilvelia** [comment]
I implemented some basic attachment downloading in 5e6a541c1ccc5e04941cd8ce0986b7afc94f23d0

In the config, it is enabled via

```json
  "attachments": {
    "download": true,
    "directory": "attachments"
  },
```

**Comment by errant-info** [comment]
Can confirm this is working nicely, downloaded over 300 attachments from a single chat (jpg, png, mov, mp4, pdf, largest file was 71mb)
